keps/sig-cluster-lifecycle/clusterapi/README.md

```diff
@@ -0,0 +1,4 @@
+# Cluster API CAEPs
+
+Cluster API follows its own process for enhancement proposals.
```

Comment:
Is there a link to a doc that explains the process?

Reply:
here it is:
https://cluster-api.sigs.k8s.io/contributing#proposal-process-caep

it can be linked from `[own process]`

Reply:
Adding!

Reply:
thanks all! that's def more helpful now :smile_cat: